### DISCUSSION TITLE
Broken hyperlinks

### DISCUSSION THREAD

**Discussion by laotzu69** [opengisch/QField#3163] [Bugs]
Hi, after the latest update the hyperlinks in my project layers' attribute tables no longer work, and are preceded  by <a href=

Could anyone help with this please?

Many thanks!

**Comment by nirvn** [comment]
@laotzu69 , this regression has been addressed in QField 2.2.3 (the actual fix commit is here https://github.com/opengisch/QField/pull/3137). It should be available on the play store within the next 48 hours.

**Comment by laotzu69** [comment]
Hi Mathieu - that's great news, many thanks for confirming.

    On Thursday, 4 August 2022 at 03:51:56 BST, Mathieu Pellerin ***@***.***> wrote:  
 
 


@laotzu69 , this regression has been addressed in QField 2.2.3 (the actual fix commit is here #3137). It should be available on the play store within the next 48 hours.
